<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-150NEV4" failures="1" tests="2" name="BankingApp Test" time="25.981" errors="0" timestamp="06 Nov 2022 05:56:28 GMT">
  <testcase classname="com.inetbanking.testCases.TC_LoginTest_001" name="loginTest" time="1.193"/>
  <testcase classname="com.inetbanking.testCases.TC_AddCustomerTest_003" name="addNewCustomer" time="15.45">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetbanking.testCases.TC_AddCustomerTest_003.addNewCustomer(TC_AddCustomerTest_003.java:53)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- addNewCustomer -->
</testsuite> <!-- BankingApp Test -->
